SYCL/Plugin/level_zero_sub_sub_device.cpp

```diff
@@ -25,89 +25,65 @@ using namespace std::chrono;
 #define INTER_NUM (150)
 #define KERNEL_NUM (2000)
 
-void run(std::vector<queue> &queues) {
+void make_queue_and_run_workload(std::vector<device> &subdevices) {
+
+  std::cout << "[important] create " << subdevices.size()
+            << " sycl queues, one for each sub-sub device" << std::endl;
```

Comment:
This is kind of misleading. How is `subdevice.size` related to sub-sub-devices?

Reply:
Agree. I've updated the variable name to reflect the same.